fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -219,6 +227,22 @@ class SceneRenderer extends WorldSystem {
         return screenSpace.unproject(this.mainCamera)
     }
 
+    /**
+     * Convert world space coordinates to screen space coordinates
+     *
+     * @param world World space coordinates
+     * @returns Pixel space coordinates
+     */
+    public WorldToPixelSpace(world: THREE.Vector3): PixelSpaceCoord {
+        this._mainCamera.updateMatrixWorld()
+        const screenSpace = world.project(this._mainCamera)
+        const widthHalf = window.innerWidth / 2
+        const heightHalf = window.innerHeight / 2
+        const x = screenSpace.x * widthHalf + widthHalf
+        const y = -(screenSpace.y * heightHalf) + heightHalf
+        return [x, y]
```

Comment:
```suggestion
        return [ window.innerWidth * (screenSpace.x + 1.0) / 2.0, window.innerHeight * (1.0 - screenSpace.y) / 2.0 ]
```

Just to reduce assignments and condense it quite a bit. This isn't something that will be constantly debugged, if ever.